.github/workflows/core.yml

```diff
@@ -7,6 +7,7 @@ on:
       - test/**
       - yt_dlp/**.py
       - '!yt_dlp/extractor/*.py'
+      - '!yt_dlp/extractor/youtube/*.py'
```

Comment:
Would changing *.py to **.py in line 9 work instead?

Reply:
Yep you're right